src/pagination/mod.rs

```diff
@@ -0,0 +1,151 @@
+use crate::cobalt_model::pagination_config::Include;
+use crate::cobalt_model::pagination_config::PaginationConfig;
+use crate::cobalt_model::permalink;
+use crate::cobalt_model::slug;
+use crate::cobalt_model::SortOrder;
+use crate::document;
+use crate::document::Document;
+use crate::error::*;
+use liquid;
+use std::cmp::Ordering;
```

Comment:
- `use liquid` shouldn't be needed anymore
- Please separate `use`s by std, external, internal

Reply:
Thank you for the review, it's done! :D